doc/version_history.rst

```diff
@@ -6,6 +6,19 @@
 Version History
 ###############
 
+v0.10.0
+------
+
+* Update Configuration schema to v4
+* Includes location in all schema
```

Comment:
I'm not sure what this means. Perhaps it would suffice to say:
```
Update `CONFIG_SCHEMA` to version v4: add a location field.
```
(the back-ticks should form a link in the build docs).